arraycontext/impl/pyopencl/fake_numpy.py

```diff
@@ -39,12 +39,15 @@
     rec_multimap_reduce_array_container,
 )
 from arraycontext.fake_numpy import BaseFakeNumpyLinalgNamespace
+from arraycontext.impl.pyopencl.taggable_cl_array import TaggableCLArray
 from arraycontext.loopy import LoopyBasedFakeNumpyNamespace
 
 
 try:
     import pyopencl as cl  # noqa: F401
     import pyopencl.array as cl_array
+
+    from arraycontext.impl.pyopencl.taggable_cl_array import TaggableCLArray
```

Comment:
`TaggableCLArray` seems to be imported twice here?

Reply:
Good catch, fixed!